ServerCore/ServerCore.csproj

```diff
@@ -25,6 +25,11 @@
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.MicrosoftAccount" Version="2.1.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="2.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.2" />
```

Comment:
Why did you add these?

Reply:
The wiki said to follow the instructions at 
https://docs.microsoft.com/en-us/ef/core/get-started/install/
to get started with EF Core.  Those instructions said to install Microsoft.EntityFrameworkCore.SqlServer and Microsoft.EntityFrameworkCore.Tools.  So I did.

I then updated the wiki here to include all those details:
https://github.com/PuzzleServer/mainpuzzleserver/wiki/Local-Development-Environment-Setup